server/config/main.go

```diff
@@ -54,16 +54,14 @@ var (
 )
 
 type Configuration struct {
-	TimeZone                string `json:"timeZone"`
-	PermissionSchemaEnabled bool   `json:"permissionSchemaEnabled"`
-	EnableErrorReporting    bool   `json:"enableErrorReporting"`
-	PluginVersion           string `json:"plugin_version"`
-
-	// derived attributes
-	BotUserID       string         `json:"botUserId"`
-	Location        *time.Location `json:"location"`
-	SentryServerDSN string         `json:"sentryServerDSN"`
-	SentryWebappDSN string         `json:"sentryWebappDSN"`
+	Location                *time.Location `json:"location"`        // derived attributes
+	BotUserID               string         `json:"botUserId"`       // derived attributes
+	SentryServerDSN         string         `json:"sentryServerDSN"` // derived attributes
+	SentryWebappDSN         string         `json:"sentryWebappDSN"` // derived attributes
```

Comment:
all these fields are present in the sequence. shouldn't we just write a single comment here?